src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -65,18 +65,22 @@
 /// Enables use of Hypre library (CMake option ENABLE_HYPRE)
 #define GEOSX_USE_HYPRE
 
+/// Macro defined when using cuda in HYPRE  (CMake option ENABLE_HYPRE_CUDA)
+#define GEOSX_USE_HYPRE_CUDA
+
 /// Enables use of PETSc library (CMake option ENABLE_PETSC)
 #define GEOSX_USE_PETSC
 
 /// Choice of global linear algebra interface (CMake option GEOSX_LA_INTERFACE)
-#define GEOSX_LA_INTERFACE Hypre
+#define GEOSX_LA_INTERFACE Trilinos
```

Comment:
Reverted back to Trilinos, intentional?

(We should look into changing the way this file is handled... it's getting out of hand, especially with more macros added)

Reply:
This seems to be a general issue. I must have built on lassen, and accidentally committed this file.